### PR TITLE
Articles/modeling/reference spec

### PR DESCRIPTION
**Pull request by casserni**

### CODE REVIEW

File: articles/modeling/reference-spec.md [hunk 1]
```diff
@@ -1 +1,44 @@
...
+
+      * Shared/Common
+
+      * External URL
+
+2.  Select a file or enter a valid url that contains data for an existing specification.
+
+3.  Select a target, if required or applicable.
+
+4.  Confirm your choice, if you see a confirm button.
```
Comment (rowa97) [inline articles/modeling/reference-spec.md]: rework, 4. Confirm your choice (Only required if ______)

File: articles/modeling/reference-spec.md [hunk 2]
```diff
@@ -1 +1,44 @@
...
+    * This Project
+
+
+      * Shared/Common
+
+      * External URL
+
+2.  Select a file or enter a valid url that contains data for an existing specification.
+
+3.  Select a target, if required or applicable.
```
Comment (rowa97) [inline articles/modeling/reference-spec.md]: specify what is or is not applicable or required

File: articles/modeling/reference-spec.md [hunk 3]
```diff
@@ -1 +1,44 @@
+# Referencing another API Specification
 
+<!-- REFBUILDER GIF/VIDEO-->
+
+## What
+
+Depending on your use case, you may find yourself needing to reference or connect to another spec or part of your current one.
+
+This can be done easily for another specification in your project, one that has been publicly shared, one from an external source, or even reference certain targets within your current specification.
```
Comment (rowa97) [inline articles/modeling/reference-spec.md]: rework, confusing

File: articles/modeling/reference-spec.md [hunk 4]
```diff
@@ -1 +1,53 @@
...
+
+      * Enter a valid **URL** to an existing specification
+
+2.  Select a **target**, if required
+
+3.  Confirm your choice. (Only required if there is a confirm button)
+
+4.  View the referenced specification by clicking the book icon
+
+## Related Links
```
Comment (rossmcdonald) [inline articles/modeling/reference-spec.md]: There should be a horizontal break here, and it should just be 'Related' in bold (not double header). See the bottom of [this](https://github.com/stoplightio/docs/blob/112b3d0da704fe3295cf97480036e6ca65e9418a/articles/testing/run-test-stoplight.md#running-a-collection) doc for reference.

File: articles/modeling/reference-spec.md [hunk 5]
```diff
@@ -1 +1,53 @@
...
+
+Documentation
```
Comment (rossmcdonald) [inline articles/modeling/reference-spec.md]: These should just be bullets, if we want to keep the 'Related' sections in the same format. Basically drop the heading sections, and just keep the link bullets.

File: articles/modeling/reference-spec.md [hunk 6]
```diff
@@ -1 +1,53 @@
+# Referencing another API Specification
```
Comment (rossmcdonald) [inline articles/modeling/reference-spec.md]: Oh and 'another' should be capitalized

**Pull request merged**